specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,294 @@
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               },
+               {  
+                  "name":"body",
+                  "in":"body",
+                  "description":"The subscription creation parameters.",
+                  "required":true,
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationParameters"
+                  }
+               }
+            ]
+         }
+      }
+   },
+   "definitions":{  
+      "SubscriptionCreationResult":{  
+         "properties":{  
+            "subscriptionLink":{  
+               "type":"string"
+            }
+         }
+      },
+      "SubscriptionCreationParameters":{  
+         "properties":{  
+            "displayName":{  
+               "type":"string"
+            },
+            "owners":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/AdPrincipal"
+               },
+               "description":"The list of principals that should be granted Owner access on the subscription."
+            },
+            "offerId":{  
+               "description":"The offer type of the subscription. For example, MS-AZR-0017P (EnterpriseAgreement) and MS-AZR-0148P (EnterpriseAgreement devTest) are available. Only valid when creating a subscription in a billing account scope.",
+               "type":"string"
+            }
+         }
+      },
+      "ErrorResponse":{  
+         "description":"Describes the format of Error response.",
+         "type":"object",
+         "properties":{  
+            "code":{  
+               "description":"Error code",
+               "type":"string"
+            },
+            "message":{  
+               "description":"Error message indicating why the operation failed.",
+               "type":"string"
+            }
+         }
+      },
+      "AdPrincipal":{  
+         "description":"Active Directory Principal for subscription creation delegated permission",
+         "properties":{  
+            "tenantId":{  
+               "description":"Tenant id of the Principal",
+               "format":"uuid",
+               "type":"string"
+            },
+            "objectId":{  
+               "description":"Object id of the Principal",
+               "format":"uuid",
+               "type":"string"
+            }
+         },
+         "required":[  
+            "tenantId",
+            "objectId"
+         ]
+      },
+      "SubscriptionOperationListResult":{  
+         "description":"A list of subscription operations.",
+         "properties":{  
+            "value":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/SubscriptionOperation"
+               }
+            }
+         }
+      },
+      "SubscriptionOperation":{  
+         "description":"status of the subscription POST operation.",
+         "properties":{  
+            "Location":{  
```

Comment:
Remove these. All we need is an "id" property for now:

        "id": {
          "type": "string",
          "description": "The operation id.",
          "readOnly": true
        }

Reply:
The Location/Retry-After should only be used in the header when we return a single subscription operation as part of a 202 response. In a 200 response when we return a SubscriptionOperation object, they don't apply.